ami/init_node.sh

```diff
@@ -39,8 +41,9 @@ sudo systemctl start elasticsearch.service
 
 # Update Kibana config to point to ElasticSearch
 echo "server.host: `curl http://169.254.169.254/latest/meta-data/public-hostname`" | sudo tee -a /etc/kibana/kibana.yml
-echo "elasticsearch.hosts: [\"http://`getent hosts firstmaster.elasticsearch | awk '{ print $1 }'`:9200\"]" | sudo tee -a /etc/kibana/kibana.yml
+echo "elasticsearch.hosts: [\"http://$masterIP:9200\"]" | sudo tee -a /etc/kibana/kibana.yml
 
```

Comment:
I initially didn't like this variable, but it should work out since the machine still always has to resolve the DNS name.  I'm looking forward to the DNS discovery process working.